packages/hash/frontend/src/pages/[shortname]/entities/[entity-uuid].page/entity-editor/properties-section/property-table/cells/value-cell/array-editor/item-limit-info.tsx

```diff
@@ -0,0 +1,59 @@
+import { Button } from "@hashintel/hash-design-system";
+import { Box, experimental_sx as sx, styled } from "@mui/material";
+import { isNumber } from "lodash";
+import { ReactNode } from "react";
+
+const StyledBox = styled(Box)(
+  sx({
+    borderRadius: 1,
+    mr: 0.5,
+    backgroundColor: "gray.20",
+    height: 26,
+    display: "flex",
+    alignItems: "center",
+    px: 1,
+    color: "gray.60",
+  }),
+);
+
+export const ItemLimitInfo = ({ min, max }: { min?: number; max?: number }) => {
+  const nodesToRender: ReactNode[] = [];
+
+  if (isNumber(min)) {
+    nodesToRender.push(
+      <>
+        <StyledBox>{min}</StyledBox> values required
+      </>,
+    );
+  }
+
+  if (isNumber(max)) {
+    nodesToRender.push(
+      <>
+        {isNumber(min) && <Box mx={0.5}>{" and "}</Box>}
+        <StyledBox>{max}</StyledBox> values maximum
+      </>,
+    );
+  }
+
+  if (!nodesToRender.length) {
+    return null;
+  }
```

Comment:
Does this not cause a keys error? Something like this is probably clearer to read. 

```suggestion
  const minNode = isNumber(min) ? (
    <><StyledBox>{min}</StyledBox> values required</>
  ) : null;
  
  const maxNode = isNumber(max) ? (
    <>
        {isNumber(min) && <Box mx={0.5}>{" and "}</Box>}
        <StyledBox>{max}</StyledBox> values maximum
      </>
  ) : null;
  
  const nodesToRender = minNode || maxNode ? <>{minNode}{maxNode}</> : null;
  
  if (!nodesToRender) {
    return null;
  }
```

Reply:
Ooops I missed this, thanks! And for the suggestion! Applied it here https://github.com/hashintel/hash/pull/1752/commits/43da9c52454845a8ef8fe83369ae662276a176da